awssdk/src/main/scala/DefaultClient.scala

```diff
@@ -291,23 +334,23 @@ class DefaultClient[F[_]: Concurrent: Timer: RaiseThrowable](
   }
 
   def batchDelete[P: Encoder](
```

Comment:
Could we add a batchDelete method that takes the items Iterable[T] and returns F[Unit], to be consistent with batchPut?

Reply:
yeah, I will do that as a separate PR. This one is large enough